src/newViews/ProjectEdit/GeoAreas/index.tsx

```diff
@@ -29,6 +31,10 @@ interface Props {
     activeProject: number;
 }
 
+function isAdminLevelGeoArea(value?: Partial<AdminLevelGeoArea>): value is AdminLevelGeoArea {
+    return !!(value as AdminLevelGeoArea)?.id;
+}
```

Comment:
What would be the different between AdminLevelGeoArea and it's Partial?
Why do we have a switch between RegionMap and RegionMapList?

Is it to show temporary map when a RegionMap is being added?